api_useradm.go

```diff
@@ -53,14 +67,39 @@ func (i *UserAdmHandlers) GetApp() (rest.App, error) {
 	return app, nil
 }
 
-func (i *UserAdmHandlers) AuthLoginHandler(w rest.ResponseWriter, r *rest.Request) {
-	rest.NotFound(w, r)
+func (u *UserAdmApiHandlers) AuthLoginHandler(w rest.ResponseWriter, r *rest.Request) {
+	l := requestlog.GetRequestLogger(r.Env)
+
+	//parse auth header
+	//TODO: we'll allow no auth in the 'first login' case, for now - just return 401
+	email, pass, ok := r.BasicAuth()
+	if !ok {
+		rest_utils.RestErrWithLog(w, r, l, ErrAuthHeader, http.StatusUnauthorized)
+		return
+	}
+
+	useradm, err := u.createUserAdm(config.Config, l)
```

Comment:
Once `NewJWTHandler..` stops returning errors and `config.Config` is removed from `UserAdmFactory` this piece becomes `useradm := u.createUserAdm(l)`